editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -413,7 +413,10 @@ struct _PaletteEntry {
 	String name;
 
 	bool operator<(const _PaletteEntry &p_rhs) const {
-		return name < p_rhs.name;
+		
```

Comment:
This line should be removed to match our current code style:
https://github.com/godotengine/godot/pull/43050/checks?check_run_id=1302112226

Could you do it by amending the commit? (`git commit --amend` and `git push --force` to update the remote branch).